Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.hxx

```diff
@@ -452,6 +452,7 @@ namespace AzToolsFramework
 
         void ResetToSlice();
 
+        static bool ShouldUseDPE();
```

Comment:
This function doesn't need to be exposed in the header.
It can be internall to the EntityPropertyEditor.cpp file as a static free function.

Reply:
done